Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Editor/WindowsMixedRealityCameraSettingsProfileInspector.cs

```diff
@@ -37,14 +37,16 @@ public override void OnInspectorGUI()
                 serializedObject.Update();
 
                 EditorGUILayout.Space();
-                EditorGUILayout.LabelField("Mixed Reality Capture Settings", EditorStyles.boldLabel);
-
                 using (new EditorGUILayout.HorizontalScope())
                 {
-                    EditorGUILayout.PropertyField(renderFromPVCameraForMixedRealityCapture, pvCameraRenderingTitle);
+                    EditorGUILayout.LabelField("Mixed Reality Capture Settings (Experimental)", EditorStyles.boldLabel);
                     InspectorUIUtility.RenderDocumentationButton(MRCDocURL);
                 }
 
+                EditorGUILayout.HelpBox("Render from PV Camera is supported on Unity 2018.4.12f1 or newer and 2019.3 or newer. Enabling the feature on other versions may result in incorrect capture behavior.", MessageType.Info);
```

Comment:
The latest fixes went into 2018.4.13f1.

There may be additional fixes in the future, but it's in much better shape at that point. Not sure exactly how we want to word that.